lib/backends/sdc/networks.js

```diff
@@ -774,7 +789,19 @@ function addNetworksToContainerPayload(opts, container, payload, callback) {
                         next(findErr);
                         return;
                     }
-                    payload.networks = [ {uuid: network.uuid, primary: true} ];
+                    payload.networks =
+                        [ {ipv4_uuid: network.uuid, primary: true} ];
+                    if (container.NetworkingConfig.EndpointsConfig[networkMode]
```

Comment:
I think its possible for us to land here without a NetworkingConfig.  I think we may need to check for null's all the way down:
```js
if (container.NetworkingConfig != null && container.NetworkingConfig.EndpointsConfig != null && container.NetworkingConfig.EndpointsConfig[networkMode] != undefined) {
```